packages/rum/README.md

```diff
@@ -2,36 +2,36 @@
 
 ## Overview
 
-Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application's individual users. To collect events, add the RUM Browser SDK to your browser application and configure what data is collected using initialization parameters.
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application's individual users. To collect events, add the Datadog Browser SDK to your browser application and configure what data is collected using initialization parameters.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why changing the name? We have two "Browser SDK", Logs and RUM. This doc is about RUM, so IMO it should be `RUM Browser SDK` (or eventually `Datadog RUM Browser SDK`)